### DISCUSSION TITLE
Creating a gene activity matrix with modified parameters

### DISCUSSION THREAD

**Discussion by JE001** [stuart-lab/signac#1255] [Q&A]
Hello. This may be a very basic question, so thank you in advance for helping me understand and find a solution to my problem. I would like to create a new gene activity assay derived from scATAC-seq data, but rather than summing the fragments intersecting the gene body and promoter region (as GeneActivity() does), I would like to assign all the fragments to the nearest gene body, regardless of their distance to it. Is there a way to change the parameters used to assign fragments to genes in the GeneActivity() function beyond just changing how far the promoter region should extend upstream?

**Comment by timoast** [comment]
This is not currently supported in the software

**Comment by YevhenAkimov** [comment]
Can you use ClosestFeature and then just summarize by feature?
